graph/tags.go

```diff
@@ -24,6 +24,9 @@ import (
 	"github.com/docker/libtrust"
 )
 
+// ErrNameIsNotExist returned when there is no image with requested name.
+var ErrNameIsNotExist = errors.New("image with specified name is not exist")
```

Comment:
`s/image with specified name is not exist/image with specified name does not exist`